fuel/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+import org.jetbrains.kotlin.gradle.dsl.Coroutines
+import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
+import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet
```

Comment:
[Q] I don't think these imports are necessary? 🤔

Reply:
Good catch. They're not necessary now, but they were in the first commits. Removed in db7772e.